R/get_proj.r

```diff
@@ -27,13 +27,6 @@ get_proj <- function(projloc=".",geteval=TRUE){
   datf  <- list.files(paste0(projloc,"/shinyMixR/app/data"))
   grepd <- " |^[[:digit:]]|\\!|\\#|\\$|\\%|\\&|\\'|\\(|\\)|\\-|\\;|\\=|\\@|\\[|\\]|\\^\\`\\{\\|\\}"
   if(any(grepl(grepd,datf))) warning("Data files with special characters found, take into acount that models that use these can crash")
-  # not relevant to read all data for running nlmixr in separate session (should be loaded in this session!)
-  lapply(list.files(paste0(projloc,"/shinyMixR/app/data"),full.names = TRUE),function(x){
-    if(!grepl(grepd,x) & !exists(sub("\\.rds$|\\.csv$","",basename(x),ignore.case = TRUE),envir=.GlobalEnv)){
-      if(grepl("\\.rds$",x,ignore.case = TRUE)) assign(sub("\\.rds$","",basename(x),ignore.case = TRUE),readRDS(x),pos = .GlobalEnv)
-      if(grepl("\\.csv$",x,ignore.case = TRUE)) assign(sub("\\.csv$","",basename(x),ignore.case = TRUE),read.csv(x),pos = .GlobalEnv)
-    }
-  })
```

Comment:
@RichardHooijmaijers I don't think we need this. I can't find any locations the app where we are pulling the data from the global environment. When we do need this data, we check if it's there, but if it's not, we pull it from the RDS files. So I think we can leave this out. At the same we get rid of the global assignments- they needed to go anyway.